examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,360 @@
+#    chunks (with overlap) and feed into pipeline.
+# 3. Collect output chunks and combine according to the way they have been
+#    overlapped.
+# 
+# The `Hybrid Demucs <https://arxiv.org/pdf/2111.03600.pdf>`__ model is a developed version of the
+# `Demucs <https://github.com/facebookresearch/demucs>`__ model, a
+# waveform based model which separates music into its
+# respective sources, such as vocals, bass, and drums. Hybrid Demucs effectively uses spectrogram to learn
+# through the frequency domain and also moves to time convolutions.
+# 
+
+
+######################################################################
+# 2. Preparation
+# --------------
+# 
+# First, we install the necessary dependencies. The first requirement is
+# ``torchaudio`` and ``torch``
+# 
+
+import torch
+import torchaudio
+
+print(torch.__version__)
+print(torchaudio.__version__)
+
+
+######################################################################
+# In addition to ``torchaudio``, ``mir_eval`` is required to perform
+# signal-to-distortion ratio (SDR) calculations. ``mir_eval`` can be
+# downloaded utilizing ``pip3`` as seen in the next cell.
+# 
+
+from IPython.display import Audio
+from torchaudio.utils import download_asset
+import matplotlib.pyplot as plt
+
+# To download mir_eval run !pip3 install mir_eval
+from mir_eval import separation
+
+try:
+    import google.colab
+
+    print(
+        """
+        To enable running this notebook in Google Colab, install nightly
+        torch and torchaudio builds and the requisite third party libraries by
+        adding the following code block to the top of the notebook before running it:
+
+        !pip3 uninstall -y torch torchvision torchaudio
+        !pip3 install --pre torch torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+        """
+    )
+except ModuleNotFoundError:
+    pass
+
+
+######################################################################
+# 3. Construct the pipeline
+# -------------------------
+# 
+# Pre-trained model weights and related pipeline components are bundled as
+# :py:func:`torchaudio.pipelines.HDEMUCS_HIGH_MUSDB_PLUS`. This is a
+# HDemucs model trained on
+# `MUSDB18-HQ <https://zenodo.org/record/3338373>`__ and additional
+# internal extra training data.
+# This specific model is suited for higher sample rates, around 44.1 kHZ
+# and has a nfft value of 4096 with a depth of 6 in the model implementation.
+
+from torchaudio.prototype import pipelines
+bundle = pipelines.HDEMUCS_HIGH_MUSDB_PLUS
+
+model = bundle.get_model()
+
+sample_rate = bundle.sample_rate
+
+print(f"Sample rate: {sample_rate}")
+
+
+######################################################################
+# 4. Configure the application function
+# -------------------------------------
+# 
+# Due to the nature of the model, it is very difficult to have sufficient
+# memory to apply the model to  an entire song at once. As a result,
```

Comment:
> As a result,

sounds like chunking is the consequence naturally happened,
but it's the technique people invented for workaround.

Something like `To workaround the limitation` sounds more natural.